src/baitapchuyentien/Account.java

```diff
@@ -0,0 +1,71 @@
+package baitapchuyentien;
+
+import baitapchuyentien.Account;
+
+public class Account {
+private String id;
+private String name;
+private int balance;
+
+//Khoi tao constructor
+
+public Account(String id, String name, int balance) {
+	this.id=id;
+	this.name=name;
+	this.balance=balance;
+}
+
+
+
+public String getId() {
+	return id;
+}
+
+public void setId(String id) {
+	this.id = id;
+}
+
+public String getName() {
+	return name;
+}
+public void setName(String name) {
+	this.name = name;
+}
+public int getBalance() {
+	return balance;
+}
+public void setBalance(int balance) {
+	this.balance = balance;
+}
+
+public void credit (int amount) {
+	this.balance +=amount;
+}
+
+public void debit (int amount) {
+	System.out.println("=======================================");
+	if (amount > this.balance) {
+		System.out.println("Thanh toan khong thanh cong");
+	}
+	else if(amount<0) {
+		System.out.println("Invalid params");
+	}
+	else {
+		this.balance -=amount;
+	}
```

Comment:
nên dùng thêm return để đỡ else nhé
```suggestion
	if (amount > this.balance) {
		System.out.println("Thanh toan khong thanh cong");
	}
	if(amount<0) {
		return System.out.println("Invalid params");
	}
	this.balance -=amount;
```